<launch>

	<node name="policy_gradient_OP" pkg="grasp_learning" type="policy_gradient_OP.py" output="screen">
 		<param name="input_output_data" value="/home/jejje/grasping_ws/src/grasping_oru/grasp_learning/stored_data/training_data/task_dynamics/task_dynamics_random_episode_1.txt" />
 	</node>
</launch>